Example/WordPress-Aztec-iOS/EditorDemoController.swift

```diff
@@ -5,20 +5,20 @@ import Aztec
 
 class EditorDemoController: UIViewController
 {
-    private var bottomConstraint: NSLayoutConstraint?
+    private var bottomConstraint: NSLayoutConstraint!
 
 
     private (set) lazy var editor: AztecVisualEditor = {
-        let e = AztecVisualEditor(textView: self.textView)
+        let e = AztecVisualEditor(textView: self.richTextView)
```

Comment:
+1 I think I did that just to inspect the variable before it was returned, but its not something that should have shipped